a.txt

```diff
@@ -0,0 +1 @@
+asdfadsf
```

Comment:
Does this file need to be checked in?

Reply:
No, no it doesn't. That's my mistake - I'll need to force push to resolve it though.